ui/apps/platform/src/Containers/VulnMgmt/Reports/Components/ScheduleText.tsx

```diff
@@ -17,7 +17,7 @@ function ScheduleText({ schedule }: ScheduleTextProps): ReactElement {
 
     const dayListType = schedule?.intervalType === 'WEEKLY' ? 'daysOfWeek' : 'daysOfMonth';
 
-    const dayList = getDayList(schedule?.intervalType, schedule[dayListType]?.days);
+    const dayList = getDayList(schedule?.intervalType, schedule[dayListType]?.days ?? []);
```

Comment:
This is the only change that impacts runtime in this PR outside of the Workload CVE updates. There doesn't appear to be a situation where `schedule[dayListType]?.days` results in `undefined`, but it it technically possible and would cause a crash if it happened.

Reply:
Outside scope of this contribution, `getDayList` is only called in Reports and is tightly coupled its types:
* better than to be exposed in utlis/dateUtils
* would be encapsulated in sub-container
    ui/apps/platform/src/Containers/VulnMgmt/Reports/VulnMgmtReport.utils.ts

Reply:
Besides understanding the scope of intended use for utils, it seems like function and associated data structures would move from unconditionally loaded main chunk to lazy loaded container chunk.